plugins/rest/rest_test.go

```diff
@@ -317,6 +317,82 @@ func TestNew(t *testing.T) {
 				awsWebIdentityTokenFileEnvVar: "TEST",
 			},
 		},
+		{
+			name: "ValidGCPMetadataIDTokenOptions",
+			input: `{
+                "name": "foo",
```

Comment:
[nit] Multiline string indentation is a little off here. I'm sure it looks OK with the proper editor settings 😉

Reply:
Now using tabs instead of spaces to match the indention of other tests.